### PR TITLE
Improve brand logo

### PR DESCRIPTION
**Pull request by zepfietje**
- [ ] Changes have been thoroughly tested to not break existing functionality.
- [ ] New functionality has been documented or existing documentation has been updated to reflect changes.
- [ ] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

File: packages/panels/src/FilamentManager.php [hunk 1]
```diff
@@ -84,11 +84,16 @@ public function getBrandName(): string | Htmlable
         return $this->getCurrentPanel()->getBrandName();
     }
 
-    public function getBrandLogo(): ?string
+    public function getBrandLogo(): string | Htmlable | null
     {
         return $this->getCurrentPanel()->getBrandLogo();
     }
 
+    public function getBrandLogoHeight(): string | int | null
```
Comment (danharrin) [inline packages/panels/src/FilamentManager.php]:
```suggestion
    public function getBrandLogoHeight(): ?string
```

File: packages/panels/src/Facades/Filament.php [hunk 2]
```diff
@@ -28,7 +28,8 @@
  * @method static string getAuthGuard()
  * @method static string | null getAuthPasswordBroker()
  * @method static string | Htmlable getBrandName()
- * @method static string | null getBrandLogo()
+ * @method static string | Htmlable | null getBrandLogo()
+ * @method static string | int | null getBrandLogoHeight()
```
Comment (danharrin) [inline packages/panels/src/Facades/Filament.php]:
```suggestion
 * @method static string | null getBrandLogoHeight()
```

**Review by danharrin — CHANGES_REQUESTED** [review]
Once you add some basic docs for `brandLogoHeight` this should be ok

File: packages/panels/resources/views/components/logo.blade.php [hunk 3]
```diff
@@ -1,19 +1,38 @@
 @php
     $brandName = filament()->getBrandName();
-    $brandLogo = filament()->getBrandLogo()
+    $brandLogo = filament()->getBrandLogo();
+    $brandLogoHeight = filament()->getBrandLogoHeight() ?? '1.25rem';
```
Comment (danharrin) [inline packages/panels/resources/views/components/logo.blade.php:4]: IMO this should be `1.5` or `1.75`

Comment (zepfietje) [reply packages/panels/resources/views/components/logo.blade.php:4]: Yeah I wasn't too sure about 1.25 vs 1.5. However, 1.25 looked slightly better with the Filament logo.

Comment (danharrin) [reply packages/panels/resources/views/components/logo.blade.php:4]: I think we should lean towards 1.5, so I am going to swap it

**Pull request merged**